<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionBrowserUi</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,1,0,10">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HListWidget" name="Type">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>61</height>
              </size>
             </property>
             <property name="currentRow">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>Vignette</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Encounter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Disposition</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Transition</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Character</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HListWidget" name="Character"/>
           </item>
           <item>
            <widget class="QLabel" name="NameLabel">
             <property name="text">
              <string>&lt;TypeName&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HListWidget" name="Name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,2">
           <item>
            <widget class="QLabel" name="DescLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;TypeDesc&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HListWidget" name="Desc"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Iteration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="HListWidget" name="Iteration"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Version</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Magnet">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Jump to current scene</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2277.png</normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2277.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>14</width>
                 <height>14</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="HListWidget" name="Work"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="WorkLabel">
           <property name="text">
            <string>Range: 0-60</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="Load">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="VersionUp">
             <property name="text">
              <string>Version Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ExportFbx">
             <property name="text">
              <string>Export fbx</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Vendor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="VendorMa">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Work</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="WorkPath">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WorkBrowser">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open browser in work dir</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2234.png</normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2234.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HListWidget</class>
   <extends>QListWidget</extends>
   <header>hlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
